frontend/src/modules/conversation/components/conversation-details.vue

```diff
@@ -0,0 +1,209 @@
+        </div>
+      </div>
+    </div>
+    <div>
+      <app-activity-content
+        :class="
+          conversation.conversationStarter.title
+            ? 'text-sm'
+            : 'text-base'
+        "
+        title-classes="text-base font-medium"
+        :activity="conversation.conversationStarter"
+        :show-more="true"
+      />
+    </div>
+    <div class="pt-6">
+      <div class="flex items-center">
+        <div class="flex items-center mr-6">
+          <i
+            class="ri-group-line text-base mr-2 text-gray-400"
+          ></i>
+          <p class="text-xs text-gray-600">
+            {{ conversation.memberCount }} participant{{
+              conversation.memberCount > 1 ? 's' : ''
+            }}
+          </p>
+        </div>
+        <div class="flex items-center">
+          <i
+            class="ri-reply-line text-base mr-2 text-gray-400"
+          ></i>
+          <p class="text-xs text-gray-600">
+            {{ conversation.activityCount - 1 }}
+            {{
+              conversation.activityCount > 2
+                ? 'replies'
+                : 'reply'
+            }}
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <div class="pt-10">
+      <app-conversation-reply
+        v-for="(reply, ri) in replies"
+        :key="reply.id"
+        :activity="reply"
+        :display-content="true"
+        :body-classes="
+          ri < replies.length - 1 ? 'pb-8' : ''
+        "
+        :show-more="true"
+      >
+        <template #underAvatar>
+          <div
+            v-if="ri < replies.length - 1"
+            class="h-full w-0.5 bg-gray-200 my-2"
+          ></div>
+        </template>
+      </app-conversation-reply>
+    </div>
+  </article>
+</template>
+
+<script>
+import AppActivityMessage from '@/modules/activity/components/activity-message'
+import AppConversationReply from '@/modules/conversation/components/conversation-reply'
+import AppActivityContent from '@/modules/activity/components/activity-content'
+import AppLoading from '@/shared/loading/loading-placeholder'
+import AppAvatar from '@/shared/avatar/avatar'
+import integrationsJsonArray from '@/jsons/integrations.json'
+import computedTimeAgo from '@/utils/time-ago'
+
+export default {
+  name: 'AppConversationDetails',
+  components: {
+    AppActivityMessage,
+    AppConversationReply,
+    AppActivityContent,
+    AppLoading,
+    AppAvatar
+  },
+  props: {
+    conversation: {
+      type: Object,
+      required: false,
+      default: () => ({})
+    },
+    loading: {
+      type: Boolean,
+      required: false,
+      default: false
+    }
+  },
+  computed: {
+    platform() {
+      console.log(this.conversation)
+      return integrationsJsonArray.find(
+        (i) => i.platform === this.conversation.platform
+      )
+    },
+    member() {
+      return this.conversation.conversationStarter.member
+    },
+    sentiment() {
+      return this.conversation.conversationStarter.sentiment
```

Comment:
Not sure if this should be 
`return this.conversation.conversationStarter.sentiment`

Instead of 
`return this.conversation.conversationStarter.sentiment.sentiment`

Reply:
it is like that just prettier put .sentiment to next line